backend/src/test/java/com/pickpick/acceptance/message/MessageAcceptanceTest.java

```diff
@@ -1,211 +1,349 @@
         // given
-        String token = jwtTokenProvider.createToken(MEMBER_ID);
-        Map<String, Object> request = createQueryParams("jupjup", "", "5", "false", "", "");
+        채널_생성_후_메시지_저장(MEMBER_SLACK_ID, ChannelFixture.QNA.create());
+        메시지_목록_생성(MEMBER_SLACK_ID, 11);
+        채널_구독_요청(token, 1L);
+
+        MessageRequestBuilder request = new MessageRequestBuilder();
 
         // when
-        ExtractableResponse<Response> response = getWithToken(MESSAGE_API_URL, token, request);
-        MessageResponses messageResponses = response.as(MessageResponses.class);
+        ExtractableResponse<Response> response = 메시지_조회(token, request);
 
         // then
         상태코드_200_확인(response);
-        assertThat(messageResponses.isNeedPastMessage()).isFalse();
+        assertThat(toMessageResponses(response).hasFuture()).isFalse();
     }
 
     @Test
-    void 이미_리마인드_완료된_메시지_조회_시_isSetReminded가_false이고_remindDate가_null() {
+    void 키워드_검색() {
         // given
-        given(clock.instant())
-                .willReturn(Instant.parse("2022-08-13T00:00:00Z"));
+        String keyword = "줍줍";
+        채널_생성_후_메시지_저장(MEMBER_SLACK_ID, ChannelFixture.QNA.create());
+        채널_구독_요청(token, 1L);
+
+        int messageCount = 5;
+        키워드를_포함한_메시지_목록_생성(MEMBER_SLACK_ID, messageCount, keyword);
 
-        String token = jwtTokenProvider.createToken(MEMBER_ID);
-        Map<String, Object> request = createQueryParams("", "", "5", "true", "", "1");
+        MessageRequestBuilder request = new MessageRequestBuilder()
+                .keyword(keyword);
 
         // when
-        ExtractableResponse<Response> response = getWithToken(MESSAGE_API_URL, token, request);
-        MessageResponse messageResponse = response.as(MessageResponses.class)
-                .getMessages()
-                .get(0);
+        ExtractableResponse<Response> response = 메시지_조회(token, request);
 
         // then
         상태코드_200_확인(response);
-        assertAll(
-                () -> assertThat(messageResponse.isSetReminded()).isFalse(),
-                () -> assertThat(messageResponse.getRemindDate()).isNull()
-        );
+        assertThat(메시지_개수(response)).isEqualTo(messageCount);
     }
 
     @Test
-    void 리마인드_해야하는_메시지_조회_시_isSetReminded가_true이고_remindDate에_값이_존재() {
+    void 특정_채널_목록에서_조회() {
         // given
-        given(clock.instant())
-                .willReturn(Instant.parse("2022-08-10T00:00:00Z"));
+        채널_생성_후_메시지_저장(MEMBER_SLACK_ID, ChannelFixture.NOTICE.create());
+        채널_생성_후_메시지_저장(MEMBER_SLACK_ID, ChannelFixture.FREE_CHAT.create());
+        채널_생성_후_메시지_저장(MEMBER_SLACK_ID, ChannelFixture.QNA.create());
 
-        String token = jwtTokenProvider.createToken(MEMBER_ID);
-        Map<String, Object> request = createQueryParams("", "", "5", "true", "", "1");
+        long[] channelIds = new long[]{1L, 2L};
+        MessageRequestBuilder request = new MessageRequestBuilder()
+                .channelIds(channelIds);
```

Comment:
그냥 궁금해서 여쭤보는건데 
```suggestion
        MessageRequestBuilder request = new MessageRequestBuilder()
                .channelIds(1L, 2L);
```
로 하지 않으신 이유가 있을까요?? 배열로 선언된걸 본게 너무 오랜만이라 정말 궁금해서 여쭤봅니다!

Reply:
channelIds의 개수를 assertThat으로 검증하는 부분에서도 사용해씩 때문입니다~~!
List로 사용하려했는데 가변 인자를 받는게 아니라 아예 List로 사용하는게 나을 것 같기도 하네요